training/templates/email_form_template.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Send Email Page</title>
+</head>
+<body>
+    {% from "macro_Form_Index.html" import render_field %}
+
+	<form method=post>
+		<dl>
+	    {{ render_field(form.email_sender) }}
+	    {{ render_field(form.email_recipient) }}
+        {{ render_field(form.message_body) }}
+        {{ render_field(form.send_now) }}
+	    <input type="submit" name="boton">
+  		</dl>
+
+		{%if sended %}
+		<p>Email enviado!</p>
+		{%endif%}
+	</form>
```

Comment:
Indentation and spacing is weird

Reply:
Yeah, have in mind that probably you will have to come back to this code and it's important that it's easy to read.